src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+        protected char taskType;
+
+        public Task(String description) {
+            this.description =  description;
+            this.isDone = false;
+        }
+
+        public String getStatusIcon() {
+            return (isDone ? "X" : " ");
+        }
+        public void markAsDone() {
+            this.isDone = true;
+        }
+
+    }
+
+    static String exitTrigger = "bye";
```

Comment:
You could try to make string constants as all caps.

Reply:
Agreed! These variables are constants and should follow the proper naming conventions